src/CFBG.cpp

```diff
@@ -782,8 +797,15 @@ bool CFBG::FillPlayersToCFBG(BattlegroundQueue* bgqueue, Battleground* bg, const
                 auto playerGuid = *((*Horde_itr)->Players.begin());
                 if (auto player = ObjectAccessor::FindConnectedPlayer(playerGuid))
                 {
-                    sumLevel += player->getLevel();
+                    auto playerLevel = player->getLevel();
+
+                    sumLevel += playerLevel;
                     sumItemLevel += player->GetAverageItemLevel();
+
+                    if (IsEnableBalanceClassLowLevel() && player->getClass() == CLASS_HUNTER && (playerLevel == bgMaxLevel || playerLevel == bgMaxLevel-1))
```

Comment:
this check is repeated twice, shouldn't we wrap it in a boolean method? so it can also have a self-descriptive name